File: app/src/main/java/za/co/woolworths/financial/services/android/ui/activities/product/dynamicyield/response/request/Context.java
package za.co.woolworths.financial.services.android.ui.activities.product.dynamicyield.response.request;

import java.io.Serializable;

public class Context implements Serializable {
    private Device device;

    public Context(Device device) {
        this.device = device;
    }

    public Device getDevice() {

        return this.device;
    }

    public void setDevice(Device device) {
        this.device = device;
    }
}

File: app/src/main/java/za/co/woolworths/financial/services/android/ui/activities/dashboard/DynamicYield/request/Context.kt
package za.co.woolworths.financial.services.android.ui.activities.dashboard.DynamicYield.request

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Context(
    val device: Device? = null,
    val page: Page? = null,
    val channel: String? = null,
    val pageAttributes: PageAttributes? = null
): Parcelable